src/components/Paginator.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const Paginator = ( {pageNumber, setPageNumber} ) => {
+    return (
+    <div className="flex flex-row justify-center items-center">
+        {pageNumber == 1 
+        ? <div>
+            <button className="invisible bg-gray-300 hover:bg-gray-500 mx-2 py-1 px-4 rounded inline-flex items-center border-8" onClick={() => setPageNumber(pageNumber - 1)}>prev</button>
+            {pageNumber}
+            <button className="bg-gray-300 hover:bg-gray-500 mx-2 py-1 px-4 rounded inline-flex items-center border-8" onClick={() => setPageNumber(pageNumber + 1)}>next</button>
```

Comment:
`border-8` makes this button pretty big, plus it's only shown on the first page.

Suggestion: put the common classes in a variable

```jsx
const buttonClasses = "invisible bg-gray-300 hover:bg-gray-500 mx-2 py-1 px-4 rounded inline-flex items-center"
<button className={buttonClasses}>prev</button>
<button className={buttonClasses}>next</button>
```